horizomer/benchmark/run_phylophlan.sh

```diff
@@ -11,22 +11,21 @@
 # usage: run PhyloPhlAn to build phylogenetic tree of whole genomes
 
 # prerequisites:
-# one needs to install the development version of PhyloPhlAn:
-#   hg clone https://bitbucket.org/nsegata/phylophlan
-#   cd phylophlan
-#   hg up dev
+# one needs to install a certain development version of PhyloPhlAn:
+#   hg clone https://bitbucket.org/nsegata/phylophlan -r 2c0e61a
+# one needs a Python 2 conda environment with biopython installed
```

Comment:
PhyloPhlAn wasn't designed to run in a conda environment. (It was designed before conda became popular.)